src/components/resource-selector/media-library/media-library-edit-screen.yml

```diff
@@ -0,0 +1,4 @@
+mediaLibraryEditScreen:
+  editHeadline:
+    en: Edit data
+    de: Datei bearbeiten
```

Comment:
The header for the upload screen is "Create media"/"Medien erstellen" and here we have "data" (=German "Daten") and "Datei" (=English "file"). Maybe we could unify the wording here. Even though we do not edit the actual media in the edit screen, I could imagine "Edit media"/"Medien bearbeiten" here. Alternatively we could use "File"/"Datei" in both?

Reply:
Ah good point and good catch 🤔  I think "Edit media" / "Medien bearbeiten" is a good working for both cases and I'm happy if we can avoid passing down a custom title to the component.
I shall change to this.

Reply:
Done (as you already know 😋 )